crates/starknet-devnet-core/src/utils.rs

```diff
@@ -112,15 +108,14 @@ pub(crate) mod test_utils {
     ) -> BroadcastedDeclareTransactionV2 {
         let contract_class = dummy_cairo_1_contract_class();
 
-        let compiled_class_hash = compute_casm_class_hash(
-            &CasmContractClass::from_contract_class(
-                contract_class.clone(),
-                true,
-                MAX_BYTECODE_SIZE_LIMIT,
-            )
-            .unwrap(),
-        )
-        .unwrap();
+        let casm_contract_class_json =
+            usc::compile_contract(serde_json::to_value(contract_class.clone()).unwrap()).unwrap();
+
+        let compiled_class_hash = serde_json::from_value::<CompiledClass>(casm_contract_class_json)
+            .unwrap()
+            .class_hash()
+            .unwrap()
+            .into();
```

Comment:
Perhaps the suggested `casm_json_to_hash` could be used in tests as well, like here and in other similar places.

Reply:
[b05f918](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/391/commits/b05f9187a30724d91fd9b78d16c59fc53a9ea601)